application/src/main/java/com/github/pieter_groenendijk/model/Loan.java

```diff
@@ -0,0 +1,101 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "loan")
+public class Loan {
+    @Id
+    @GeneratedValue(strategy = jakarta.persistence.GenerationType.IDENTITY)
+    private long loanId;
+    @Column(name = "startDate", nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date startDate;
+    @Column(name = "endDate", nullable = true)
+    @Temporal(TemporalType.DATE)
+    private Date endDate;
+    @Column(name = "returnDate", nullable = true)
```

Comment:
`endDate` is de datum waarop het boek maximaal ingeleverd mag worden zonder boete.
`returnDate` is de datum waarop die in werkelijkheid is ingeleverd.

Klopt dit? Ik denk persoonlijk dat de naamgeving duidelijker zou kunnen. zoals bijv. : `returnBy` en `returnedOn`.